docker/beacon.Dockerfile

```
@@ -1,19 +1,18 @@
 FROM python:3.7
-
+# Set up code directory
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
 # Install deps
 RUN apt-get update
 RUN apt-get -y install libsnappy-dev gcc g++ cmake
 
-ARG GITREF=interop
-
 RUN git clone https://github.com/ethereum/trinity.git .
-RUN git checkout $GITREF
-RUN pip install -e .[dev] --no-cache-dir
+RUN pip install -e .[eth2-dev] --no-cache-dir
```

We might want milagro-bls-binding in 'eth2-extra'
```suggestion
RUN pip install -e .[eth2-dev,eth2-extra] --no-cache-dir
```